index.js

```diff
@@ -8,12 +8,14 @@ const cors = require('cors')
 const passport = require('passport')
 
 const mongoose = require('mongoose')
-const mongoUri = process.env.DATABASE_URL || 'mongodb://localhost/ufood'
+const mongoUri = process.env.DATABASE_URL || 'mongodb://localhost:27017/ufood'
 mongoose.connect(mongoUri, {
   autoReconnect: true,
   useNewUrlParser: true
 })
 
+connectWithRetry()
```

Comment:
Why this call?

Reply:
Oops. I had some issues trying to make the api work with docker on Windows. This was part of my test code. I thought I had gotten rid off of it.